### PR TITLE
The check's actual result property is now set to be the audit command…

### LINKED ISSUE #280: Actual value is empty
The "actual_value" field in the program's output is empty

### PR DESCRIPTION
**Pull request by yoavAqua**
…'s output

fix #280

### CODE REVIEW

File: check/test.go [hunk 1]
```diff
@@ -230,7 +229,7 @@ func (ts *tests) execute(s string) *testOutput {
 	}
 
 	finalOutput.testResult = result
-	finalOutput.actualResult = res[0].actualResult
+	finalOutput.actualResult = s
```
Comment (lizrice) [inline check/test.go]: This looks strange to me - isn't `s` the audit command to be executed?

Comment (yoavAqua) [reply check/test.go]: it's the output of the audit command

Comment (lizrice) [reply check/test.go]: Ah, OK... i can't help thinking that the name `execute()` is pretty bad for these functions - more like `checkResults()`. 

On the occasions where we call `failTestItem()` the string is currently being set to an error message explaining what's wrong (e.g. bad YAML). I think with this change we are going to lose those messages as they're going to get overwritten by the original string. Wdyt?

Comment (yoavAqua) [reply check/test.go]: True, fixing it now

**Comment by yoavAqua** [comment]
@lizrice Can you please review? I fixed the issue with overriding the failTestItem() output by first checking that the actual result is empty.

**Pull request merged**